space_test.go

```diff
@@ -278,12 +278,13 @@ func minimumRequiredCreateSpace() *app.CreateSpacePayload {
 	}
 }
 
-func CreateSpacePayload(name string) *app.CreateSpacePayload {
+func CreateSpacePayload(name, description string) *app.CreateSpacePayload {
```

Comment:
@kwk Yeah, that sounds like a good idea. Want to make a tracker issue? (wouldn't set it as important yet but.. :+1:  )